packages/node_modules/@ciscospark/react-component-spinner/src/index.js

```diff
@@ -1,8 +1,18 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+
 import classNames from 'classnames';
 
 import styles from './styles.css';
 
-export default function spinner() {
-  return <div className={classNames(`ciscospark-spinner`, styles.spinner)} />;
+export default function spinner({bright}) {
+  const classes = [`ciscospark-spinner`, styles.spinner];
+  if (bright) {
+    classes.push(styles.bright);
+  }
+  return <div className={classNames(classes)} />;
```

Comment:
TIL classnames can be flagged, so another way to do this check is:
`classNames('ciscospark-spinner', styles.spinner, {[styles.bright]: bright})`